packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -509,13 +537,26 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
 
     if (!ev.defaultPrevented &&
       this.props.menuTriggerKeyCode !== null &&
-      ev.which === (this.props.menuTriggerKeyCode === undefined ? KeyCodes.down : this.props.menuTriggerKeyCode)) {
+      this._isValidMenuOpenKey(ev)) {
       this._onToggleMenu();
```

Comment:
as part of _onToggleMenu could we set focus to the container before setting the state to toggle the menu?

Reply:
Would that be what we want to do? By doing this we would force the focus to go back to the menu in every scenario.

From my understanding, we want focus to go back to the previous focused element, wIth the suggested change, we always focus on the button when we close the menu.

Reply:
The focus should always only be on the container (even if it was fired against the split button portion). We should never be setting focus on the individual portions of the splitButton

Reply:
Added the changes here. Now we only don't focus the split button if everything is disabled.